common/src/main/scala/explore/Enums.scala

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore
+
+import cats.effect.IO
+import cats.syntax.all.*
+import clue.StreamingClient
+import explore.model.enums.ObsAttachmentTypeMeta
+import lucuma.core.util.Display
+import lucuma.core.util.Enumerated
+import lucuma.schemas.ObservationDB
+import lucuma.schemas.ObservationDB.Enums.ObsAttachmentType
+import queries.common.EnumQueriesGQL
+
+case class Enums(enumMeta: Enums.Meta) {
+
+  // The givens are apparently (probably) constructed lazily.
+  // See https://alexn.org/blog/2022/05/11/implicit-vs-scala-3-given/
+  // We want to fail immediately if there is a problem, so we'll reference
+  // the enumerated givens here. Add any new enums to this list.
+  Enumerated[ObsAttachmentType]
+  Enumerated[ObsAttachmentTypeMeta]
+
+  given Enumerated[ObsAttachmentTypeMeta] = Enumerated
+    .from(
+      enumMeta.obsAttachmentType.head._2,
+      enumMeta.obsAttachmentType.tail.map(_._2).toList*
+    )
+    .withTag(_.tag.show)
+
+  given Enumerated[ObsAttachmentType] = Enumerated
+    .from(
+      enumMeta.obsAttachmentType.head._1,
+      enumMeta.obsAttachmentType.tail.map(_._1).toList*
+    )
+    .withTag(_.show)
```

Comment:
What would you do for a case that requires distinguishing the enum values for logic in the code?  For example, if you need to treat attachment type `Foo` differently than `Bar` in some context?

Reply:
We can use the clue generated enums. See the `ObsAttachmentType` in `ObsAttachmentTypeMeta.scala`

Reply:
Or, we could do that with additional properties.